### ISSUE TITLE
FIBO: Possibly inconsistent updating of residual in presence of volatile oil

### ISSUE THREAD

**Issue by bska** [OPM/opm-autodiff#267]
Commit ed02b4a9 (PR #74) added initial support for volatile oil ("wet gas") to class `FullyImplicitBlackoilSolver`.  In so doing, method `computeAccum(state)` got the following treatment
```diff
             rq_[pg].accum[aix] += state.rs * rq_[po].accum[aix];
+            rq_[po].accum[aix] += state.rv * rq_[pg].accum[aix];
```
This sequence is still present in the current version of method `computeAccum(state)` in the implementation file [`opm/autodiff/FullyImplicitBlackoilSolver_impl.hpp`](https://github.com/OPM/opm-autodiff/blob/master/opm/autodiff/FullyImplicitBlackoilSolver_impl.hpp).

Is this correct?  Doesn't this lead to `rq_[po]` (oil residual) being wrong in systems that feature both dissolution and evaporation?  I would have thought that we needed to introduce a copy of
```
rq_[pg].accum[aix]
```
prior to updating that quantity in order to produce a correct update of `rq_[po]`.

Comments?

**Comment by totto82** [comment]
I agree, this is a bug. The "wet gas" option has only been tested without "live oil" present. (As in the SPE 3 case). I have tested the effect of fixing this, by adding a copy as you suggested, on  the Norne case where both "wet oil" and "live gas" is present. Fixing this has an effect on the solution, but the effect is not significant. 

Thank you for noticing. @osae is currently testing the combination of wet gas and live oil on simple 1D cases to see if there are other similar bugs when wet gas and live oil is combined.

**Comment by bska** [comment]
Closing as resolved in PR #269.
